controllers/articles/index.js

```diff
@@ -530,5 +530,30 @@ export default {
         error: 'Something went wrong'
       });
     }
+  },
+
+  getBookmarkedArticles: async (req, res) => {
+    const userId = req.user.id;
+
+    try {
```

Comment:
nice work on the PR. however, I feel since you're using find and count all implement pagination.
outside that nice job.

Reply:
ok. thanks for pointing that out